assets/cubyz/shaders/skybox/fragment.fs

```diff
@@ -0,0 +1,179 @@
+#version 330 
+
+layout (location=0) out vec4 fragColor;
+
+in vec3 pos;
+
+#define PI 3.141592
+#define iSteps 16
+#define jSteps 8
+#define ATMOS_SIZE 6371e3
+
+uniform mat4 lightDir;
+uniform mat4 invLightDir;
+uniform float altitude;
+
+vec2 rsi(vec3 r0, vec3 rd, float sr) {
+	float a = dot(rd, rd);
+	float b = 2.0 * dot(rd, r0);
+	float c = dot(r0, r0) - (sr * sr);
+	float d = (b*b) - 4.0*a*c;
+	if (d < 0.0) return vec2(1e5,-1e5);
+	return vec2(
+		(-b - sqrt(d))/(2.0*a),
+		(-b + sqrt(d))/(2.0*a)
+	);
+}
+
+vec3 atmosphere(vec3 r, vec3 r0, vec3 pSun, float iSun, float rPlanet, float rAtmos, vec3 kRlh, float kMie, float shRlh, float shMie, float g) {
```

Comment:
What are those names? What is a `shMie`?
And no, copying code is no excuse for shitty naming.